### ISSUE TITLE
trying to build this yields unsatisfiable constraints

### ISSUE THREAD

**Issue by bwl21** [dc-uba/docker-alpine-texlive#1]
Hi searching for a minimal latex container, I found your work.  I want to extend it somehow but did not even manage to create the container ...

running `docker build -t foo .` yields

```

ERROR: unsatisfiable constraints:
  perl-5.24.4-r1:
    breaks: world[perl=5.24.0-r0]
  wget-1.18-r4:
    breaks: world[wget=1.18-r2]
```

**Comment by ivanpondal** [comment]
Hello @bwl21 ! It's been a long time since I last played with this project, however the error you encountered is easy to fix. The only thing you need to do is remove the version constraints from the `apk add` command.

**Comment by bwl21** [comment]
Thanks, I will try this.

**Comment by ivanpondal** [comment]
Fixed in #2
